project_templates/fastapi_safir_app/templatekit.yaml

```diff
@@ -17,6 +17,10 @@ dialog_fields:
   - label: "Initial copyright holder"
     key: "copyright_holder"
     component: "select"
+  - label: "UWS service"
```

Comment:
We’re still using the dialog API from Slack, which limits us to five fields in a dialog. I’d suggest two changes to `templatekit.yaml` and the cookiecutter variables to accommodate that well:

1. Drop the “Deployed with Helm” field from templatekit.yaml. You can either then remove all the associated Kustomize templates, or for now just leave in cookiecutter.yaml a True default for `uses_helm`.

2. Consider reconceptualizing the `UWS Service` flag field into a `Flavor` or `Kind` choice. The default would be “Regular” and the second choice would be “UWS”. This will let us pack more of these app variations into the same Slack dialog field. Of course we don’t have to do this now since there aren’t any other variations.

Reply:
Oh, great idea, thank you! Helm removal now submitted as https://github.com/lsst/templates/pull/262. When that's merged, I'll rebase and change this to a flavor option instead.

Reply:
This is now done, and there were no changes to the generated examples so I think I did the transformation correctly.